docs/src/index_cn.tmpl

```diff
@@ -72,8 +72,8 @@ function change(el)
 <li><a onclick="return toggle('sec1')"><img id="sec1_image" src="plus.png" style="border:0;margin-right:5px;vertical-align:middle;" />Configuration Wizards</a>
 	<div id="sec1" style="display:none;">
 		<UL>
-			<LI><A class="tooltips" HREF="config/stepconf-cn.html">Stepper Configuration Wizard<span>For Parallel Port Stepper Machine</span></A>
-			<LI><A class="tooltips" HREF="config/pncconf-cn.html">Mesa Configuration Wizard<span>For Mesa Servo &amp; Stepper Machines</span></A>
+			<LI><A class="tooltips" HREF="config/stepconf_cn.html">Stepper Configuration Wizard<span>For Parallel Port Stepper Machine</span></A>
+			<LI><A class="tooltips" HREF="config/pncconf_cn.html">Mesa Configuration Wizard<span>For Mesa Servo &amp; Stepper Machines</span></A>
```

Comment:
All four of these files (`{step,pnc}conf{-,_}cn`) don't exist - just another issue with the Chinese translation.  For now the links should point to the english (untranslated) versions.